### ISSUE TITLE
Default Console Logger does not respect the showDataTime configuration

### ISSUE THREAD

**Issue by fritogotlayed** [net-commons/common-logging#79]
When using the default logger an error exists where the timestamp is always output. I tried using an empty string and space in the `dataTimeFormat` configuration. For empty string the default `DateTime.ToString()` was used. That was expected per the documentation if the `showDataTime` flag was `true`. Using a space cause an exception to be raised. As a side note it may be worth using `string.IsNullOrWhitespace(...)`. This wouldn't cause any issues if the format contained white space for vaild formats. 

### Configuration tested with
```xml
<?xml version="1.0" encoding="utf-8" ?>
<configuration>
  <configSections>
    <sectionGroup name="common">
      <section name="logging" type="Common.Logging.ConfigurationSectionHandler, Common.Logging" />
    </sectionGroup>
  </configSections>
  <startup> 
    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
  </startup>
  <common>
    <logging>
      <factoryAdapter type="Common.Logging.Simple.ConsoleOutLoggerFactoryAdapter, Common.Logging">
        <arg key="level" value="ALL" />
        <arg key="showLogName" value="false" />
        <arg key="showDataTime" value="false" />
        <arg key="dateTimeFormat" value="yyyy/MM/dd HH:mm:ss:fff" />
      </factoryAdapter>
    </logging>
  </common>
</configuration>
```
### Test code
```csharp
var log = LogManager.GetLogger<object>()
log.Debug("Test Message.");
```

#### Screen shot
![image](https://cloud.githubusercontent.com/assets/5406580/6090419/3dbe4e1a-ae3e-11e4-9ccb-8270fa629497.png)

#### Expected Result
When `showDataTime` is set to `false` the console does not write the time stamp out but does write the message.

**Comment by sbohlen** [comment]
Thanks for reporting this; I'll try to take a look at it shortly.  If you'd like to expedite a fix, we'd be happy to consider a PR too of course ...

**Comment by fritogotlayed** [comment]
@sbohlen Thanks for accepting the bug! I was going to make a PR but I just haven't had a chance to even crack the source yet. I figured I would take a swing at it tonight when I got home. :-)

**Comment by sbohlen** [comment]
Excellent; if you have any troubles, please let me know.  I'm probably not going to be able to dig into this until next weekend at the soonest, given the way this week's schedule is shaping up.

**Comment by fritogotlayed** [comment]
@sbohlen I peeked at the code a bit and I think this may be a documentation fail. The [documentation on sourceforge](http://netcommon.sourceforge.net/docs/2.1.0/reference/html/ch01.html#logging-declarative-config) states the configuration key is `showDataTime` but the code is looking for `showDateTime`. I personally feel that `showDateTime` is a better representation for what is actually happening. Judging from your title change earlier I assume you will feel that way as well :smile:. 

I changed my test around to use "Date" instead of "Data" and the logging worked as described above. I'm good with closing this bug if you are. It might be time to update the documentation so someone else doesn't fall into the same trap. The only reason I even reported the bug is that having two date times output in my build logs finally bugged me enough to look up how to change it... 

Thanks a ton again for your responsiveness on this!
